mllib/src/main/resources/org/apache/spark/ml/feature/stopwords/english.txt

```diff
@@ -0,0 +1,319 @@
+a
```

Comment:
If other lists are from NLTK, maybe we should use their English stopwords too. It would be good to make quick comparison and check the differences.